src/Test/L0/RegexSecretL0.cs

```diff
@@ -4,12 +4,13 @@
 
 namespace Microsoft.VisualStudio.Services.Agent.Tests
 {
-    public class RegexMaskL0
+    public sealed class RegexSecretL0
     {
+        // TODO: Fix these test names.
```

Comment:
Added a TODO. On failure, Xunit already prints the method name qualified with the full class name. These method names should be fixed to follow the "It <description>" pattern.